Include/fileutils.h

```diff
@@ -19,6 +19,17 @@ PyAPI_FUNC(char*) Py_EncodeLocale(
 
 PyAPI_FUNC(PyObject *) _Py_device_encoding(int);
 
+#if defined(MS_WINDOWS) || defined(__APPLE__)
+/* On Windows, the count parameter of read() is an int
+   See issue #24658
```

Comment:
This makes it sound like Windows is dealt with by bpo-24658, but it's actually macOS in that issue.  There should be some brief text added about macOS.  If you can dig up an issue number about Windows, that could be added as well.

Reply:
fixed: bpo numbers are now clarified